src/integrationTest/groovy/wooga/gradle/snyk/tasks/SnykTaskIntegrationSpec.groovy

```diff
@@ -105,36 +106,36 @@ abstract class SnykTaskIntegrationSpec<T extends SnykTask> extends SnykIntegrati
-        "executable" | toSetter(property)      | "snyk4"                      | _           | "Provider<String>"
-
-        "snykPath"   | toProviderSet(property) | osPath("/path/to/snyk_home") | _           | "File"
-        "snykPath"   | toProviderSet(property) | osPath("/path/to/snyk_home") | _           | "Provider<Directory>"
-        "snykPath"   | toSetter(property)      | osPath("/path/to/snyk_home") | _           | "File"
-        "snykPath"   | toSetter(property)      | osPath("/path/to/snyk_home") | _           | "Provider<Directory>"
-
-        "insecure"   | toProviderSet(property) | true                         | _           | "Boolean"
-        "insecure"   | toProviderSet(property) | false                        | _           | "Provider<Boolean>"
-        "insecure"   | toSetter(property)      | true                         | _           | "Boolean"
-        "insecure"   | toSetter(property)      | false                        | _           | "Provider<Boolean>"
-
-        "debug"      | toProviderSet(property) | true                         | _           | "Boolean"
-        "debug"      | toProviderSet(property) | false                        | _           | "Provider<Boolean>"
-        "debug"      | toSetter(property)      | true                         | _           | "Boolean"
-        "debug"      | toSetter(property)      | false                        | _           | "Provider<Boolean>"
+        property         | method                  | rawValue                     | returnValue | type
+        "token"          | toProviderSet(property) | "some_token"                 | _           | "String"
+        "token"          | toProviderSet(property) | "some_token"                 | _           | "Provider<String>"
+        "token"          | toSetter(property)      | "some_token"                 | _           | "String"
+        "token"          | toSetter(property)      | "some_token"                 | _           | "Provider<String>"
+
+        "logFile"        | toProviderSet(property) | osPath("/path/to/logFile")   | _           | "File"
+        "logFile"        | toProviderSet(property) | osPath("/path/to/logFile")   | _           | "Provider<RegularFile>"
+        "logFile"        | toSetter(property)      | osPath("/path/to/logFile")   | _           | "File"
+        "logFile"        | toSetter(property)      | osPath("/path/to/logFile")   | _           | "Provider<RegularFile>"
+
+        "executableName" | toProviderSet(property) | "snyk1"                      | _           | "String"
+        "executableName" | toProviderSet(property) | "snyk2"                      | _           | "Provider<String>"
+        "executableName" | toSetter(property)      | "snyk3"                      | _           | "String"
+        "executableName" | toSetter(property)      | "snyk4"                      | _           | "Provider<String>"
```

Comment:
Why the rename?

Reply:
I think the expectation in gradle API is for a property named executable to refer either to a full path; in our context we use it only when composing that.